src/main/scala/mesosphere/mesos/Availability.scala

```diff
@@ -0,0 +1,25 @@
+package mesosphere.mesos
+
+import java.util.concurrent.TimeUnit
+
+import org.apache.mesos.Protos.Offer
+
+object Availability {
+
+  def offerAvailable(offer: Offer, drainingTime: Long): Boolean = {
```

Comment:
Yes, IMO better changing just name. This way this would be the first variable of MarathonConf to have the time unit in its name 🎆 
Some examples are mesosHeartbeatInterval, onElectedPrepareTimeout, scaleAppsInterval, scaleAppsInitialDelay. All are time units but in Long and without time unit in the name.